homeassistant/components/light/hue.py

```diff
@@ -155,6 +160,12 @@ def unthrottled_update_lights(hass, bridge, add_devices):
 
     if new_lights:
         add_devices(new_lights)
+        for light in new_lights:
+            dispatcher_send(
```

Comment:
I don't think you should call the dispatcher here.

Reply:
Ah I see now, after I added the others calls. I think you're right in that I've just created the `HueLight` object with its info, so there's no reason to update, right? I'll retest without.